### PR TITLE
Hw 3

### PR DESCRIPTION
**Pull request by captain2b**
feels like error handling could be better??

### CODE REVIEW

**Comment by captain2b** [comment]
Ups exposed a secret - one sec i'll fix it

**Comment by captain2b** [comment]
> 

fixed

File: src/types/user.interfaces.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    isDeleted: boolean;
+}
+
+export interface UserMapper {
+    toDomain(x: UserDal): User;
+    toDalEntity(x: User): UserDal;
+}
+
+export interface UserServiceInterface {
+    model:  any;
```
Comment (captain2b) [inline src/types/user.interfaces.ts]: still thinking

Comment (captain2b) [reply src/types/user.interfaces.ts]: done

**Review by farafonoff — APPROVED** [review]

**Pull request merged**